components/mcb/Core/Inc/mcb.h

```diff
@@ -61,7 +61,7 @@
  */
 #define ADC_LOWER_DEADZONE            900
 #define ADC_FOR_NO_SPIN               1300    
```

Comment:
Based on the data you showed I would go for 1350-1400. The reason being is that the lower line that your data showed **I am assuming** means they were completely off the pedal. Therefore having `ADC_FOR_NO_SPIN` be 1330 would mean that the pedal just sitting there could result in acceleration because of noise. **We want a threshold outside of this noisy region**. In your picture it seems that you chose the lowest point of the noise so I would actually take the highest point of the noise and add 20 to it and make that the `ADC_FOR_NO_SPIN` value. As a guess based on what you showed something like 1375 might be good.